src/components/DataBrowser.vue

```diff
@@ -324,21 +316,21 @@ girder-data-table.girder-file-browser(
               ref="breadcrumb",
               :location="location_",
               @crumbclick="changeLocation",
-              :no-root="noRoot")
+              :root-location="!noRootLocation")
           v-spacer
           slot(name="headerwidget")
           v-btn.ma-0(flat,
               small,
               color="secondary darken-2",
-              v-if="newFolderEnabled && nonRootLocation",
+              v-if="newFolder && isNonRootLocation",
```

Comment:
I'll remove these buttons completely in the next PR -- that's what we decided on right?

Reply:
Yes

Reply:
But you will port them to the Concrete version, right?

Reply:
Yep!